### ISSUE TITLE
Changing Keyboard Configuration

### ISSUE THREAD

**Issue by JesusPresley** [pideck/pideck#139]
Hey, really nice Project!!!
I'd like to use pideck 'headless', just with a simple Usb-Numpad. Is there a way to reconfigure the F-Keys to simple Number Keys?
Let's pretend I'm a Linux Noob...

**Comment by danielhjames** [comment]
Yes, you can modify the xwax source code to have any keyboard shortcuts you like, but if you use the regular numbers 1 to 9 this will clash with the search feature, so for example you can't search for '3 Feet High and Rising' without triggering the action mapped to the 3 key. If you don't care about the search feature, you can disable it in the xwax interface.
